src/laser/MultiLaser.cpp

```diff
@@ -1162,15 +1162,20 @@ MultiLaser::InitLaserSlice (const int islice, const int comp)
             }
             else if (laser.m_laser_init_type == "gaussian") {
                 const amrex::Real a0 = laser.m_a0;
-                const amrex::Real w0 = laser.m_w0;
+                const amrex::Real w0_2 = laser.m_w0 * laser.m_w0;
                 const amrex::Real cep = laser.m_CEP;
                 const amrex::Real propagation_angle_yz = laser.m_propagation_angle_yz;
-                const amrex::Real PFT_yz = laser.m_PFT_yz - MathConst::pi/2.0;
                 const amrex::Real x0 = laser.m_position_mean[0];
                 const amrex::Real y0 = laser.m_position_mean[1];
                 const amrex::Real z0 = laser.m_position_mean[2];
                 const amrex::Real L0 = laser.m_L0;
+                const amrex::Real inv_tau2 = 1/(laser.m_tau*laser.m_tau);
                 const amrex::Real zfoc = laser.m_focal_distance;
+                const amrex::Real zeta = laser.m_zeta;
+                const amrex::Real beta = laser.m_beta;
+                const amrex::Real phi2 = laser.m_phi2;
+                const amrex::Real clight = PhysConstSI::c;
```

Comment:
Please check how this is done in the rest of the code. Either make sure it works in normalised units, or ASSERT somewhere that only SI works.